test/CadlRanchMockApis/src/FirstTest-TypeSpec.ts

```diff
@@ -1,100 +0,0 @@
-import { passOnSuccess, ScenarioMockApi, mockapi, json } from "@azure-tools/cadl-ranch-api";
-import { ValidationError } from "@azure-tools/cadl-ranch-api";
-
-/**
- * Test mock server for `FirstTest-TypeSpec` test project.
- */
-export const Scenarios: Record<string, ScenarioMockApi> = {};
-
-Scenarios.FirstTest_CreateLiteral = passOnSuccess([
```

Comment:
So, 6 operations are deleted. You should delete the corresponding 6 operations in test\TestProjects\FirstTest-TypeSpec\FirstTest-TypeSpec.tsp